packages/theme/components/FormComponents/EmailInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <SfInput
+      @click="e => $emit('click', e)"
+      @input="e => $emit('input', e)"
+      v-bind="{...$attrs, ...$props}"
+      :class="{'disable-input': disabled}"
+      :valid="!errors[0]"
+      :errorMessage="errors[0]"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfInput
+} from '@storefront-ui/vue';
+
+export default {
+  name: 'EmailInput',
+  props: {
+    disabled: {
```

Comment:
Make sure which props are set by default to the values we consider "default". If our value doesn't differ from the component's default value, we should remove it from our props definition. Try to keep FormComponents as minimal as possible (while being open for further customization).

Inside the `SfInput` component, the `disabled` property is set to false by default. Hence, our property should be removed.

https://docs.storefrontui.io/?path=/docs/components-atoms-input--common

Reply:
That's a good point. However, as mentioned above, we can't completely remove the props definition because props aren't being inherited by `SfInput`. We can only remove the `default` of those props.